test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -63,11 +118,19 @@
     private TestingServer zkTestServer = null;
     private ControllerWrapper controllerWrapper = null;
     private Controller controller;
+    private ControllerServiceImpl controllerService;
```

Comment:
You already have Controllers instantiated here. You don't need another one.

Reply:
Yes, I do have Controller instance but ControllerService instance is also required here as this has overridden methods of creating, deleting and updating KVTables along with ResponseObserver<> parameter which I can use in my tests to get the return status for the requested APIs.